app/models/batch.rb

```diff
@@ -24,7 +24,7 @@ def trigger_webhook
     return if webhook_triggered
 
     WebhookWorker.perform_async(
-      BatchPresenter.new(self).report,
+      BatchPresenter.new(self).report.to_json,
```

Comment:
I'm not sure what `report` looks like but if it's a hash this will end up sending a hash as a json string and we'd end up potentially losing some information.

Reply:
That's ok, the first line of the worker was to convert the hash to a json string (https://github.com/alphagov/link-checker-api/pull/616/files#diff-99ed8256ef1ae32bd4f66dbc72fa050ef6f2bbd611bb774d66877219b52f34c0L22) and I've removed that now.

So previously: hash sent through, then converted to a json string as first operation, then continue working with json string
Now: json string sent through, worker utilises that.